pkg/bg/bg.go

```diff
@@ -60,3 +62,17 @@ func logError(err error, cfg config) {
 
 	cfg.logger.Error().Err(err).Str("worker.name", name).Msgf("Background task failed")
 }
+
+// printStack prints the stack trace when a panic occurs
+func printStack() {
+	buf := make([]byte, 1024)
+	for {
+		n := runtime.Stack(buf, false)
+		if n < len(buf) {
+			buf = buf[:n]
+			break
+		}
+		buf = make([]byte, 2*len(buf))
+	}
+	fmt.Printf("Stack trace:\n%s\n", string(buf))
```

Comment:
nit: should we move stack to:?
```go
cfg.logger.Error().Err(err).Str("worker.name", name).Intefrace("stack", stack).Msgf("Background task failed")
```

Reply:
https://github.com/zeta-chain/node/pull/2465/commits/36435d0f22541fecc64a61639d05757f8e6a7477

Reply:
We already this this log line in `logError(...)`; no need to emit logs twice

Reply:
https://github.com/zeta-chain/node/pull/2465/commits/f199256494cbb920ae8f242be7b0587363eb9d1e